pi5/board.go

```diff
@@ -82,15 +80,17 @@ func newBoard(
 
 		// store addresses + other stuff here
 		gpioNodePath: "",
-		mem_fd:       0,
-		phys_addr:    &physical_address,
-		virt_addr:    &virtual_address,
-		gpiomem_idx:  gpiomem_idx,
+		physAddr:     INVALID_ADDR,
```

Comment:
I thought we were removing these? be sure to merge again with your first branch (or main once you merge it)

Reply:
yep will do!